scripts/zones/Sacrarium/Zone.lua

```diff
@@ -62,4 +62,47 @@ end
 zoneObject.onEventFinish = function(player, csid, option, npc)
 end
 
+zoneObject.onZoneWeatherChange = function(weather)
+    local elel = GetMobByID(ID.mob.ELEL)
+    local vanadielHour = VanadielHour()
+
+    if
+        elel and
+        elel:getZone():getLocalVar('elelQueued') == 0 and -- Why doesn't onZoneWeatherChange contain the zone object...?
```

Comment:
man if I had a nickel every time I ran into that and asked the same thing...

I'd have 2. which isn't a lot but annoying that it happened twice right?